MapboxNavigation/MGLMapView+MGLNavigationAdditions.h

```diff
@@ -1,3 +1,4 @@
+#pragma clang diagnostic ignored "-Wincomplete-implementation"
```

Comment:
This silences the diagnostic for every file that includes this header and thus every project that includes this SDK. Use `clang diagnostic push` and `clang diagnostic pop` to scope this pragma to just the lines that need it. See [this article](http://nshipster.com/clang-diagnostics/) for more information about diagnostic pragmas.